tests/tests.js

```diff
@@ -726,17 +740,17 @@ function codecTests(manifestUrl, format) {
     const pipelineConfigDict = {
       'streaming_mode': 'vod',
       'resolutions': [],
+      'channel_layouts': ['stereo'],
       'audio_codecs': ['aac', 'opus'],
     };
     await startStreamer(inputConfigDict, pipelineConfigDict);
     await player.load(manifestUrl);
```

Comment:
No need to call player.load() if you're using getAudioAndVideoCodecs(), which will do that internally.